Dockerfile.cuda12

```diff
@@ -0,0 +1,30 @@
+FROM nvidia/cuda:12.1.1-cudnn8-devel-ubuntu22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /
+
+RUN apt update -y && apt install -y ffmpeg \
+        libsm6 \
+        libxext6 \
+        curl \
+        ca-certificates \
+        sudo \
+        git \
+        python3.10 \
+        python3-pip \
+        python-setuptools \
+        python3-wheel \
+        python3-opencv \
+        xorg \
+        nvidia-driver-460
+
+RUN rm /usr/bin/python3 && ln -s python3.10 /usr/bin/python3
+
+RUN python3 -m pip install -U pip && python3 -m pip install -U setuptools
+
+RUN git clone https://github.com/danbider/lightning-pose
```

Comment:
I don't think we need to clone if we are running from the folder that already has lightning-pose. Besides this could break versioning.

Reply:
switched to copying the folder!

Reply:
also had to make some changes because of the ubuntu version (22.04 v. 18.040 so the Dockerfile.cuda12 structure is different.